### PR TITLE
Moved account selector to settings, renamed files

### PR DESCRIPTION
**Pull request by mikeger**
- Renamed team selector to account selector.
- Moved account selector to settings.
- Added team image to the conversation list.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/SelfProfile/AccountSelectorController.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+        
+        setShowTeams(to: SessionManager.shared?.accountManager.accounts.count > 1)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private var showTeams: Bool = false
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/SelfProfile/AccountSelectorController.swift]: Should we also replace the usage of `Team`with `Account` in this class?

File: Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift [hunk 2]
```diff
@@ -275,14 +278,22 @@ public final class PersonalTeamView: BaseTeamView {
...
         self.accessibilityTraits = UIAccessibilityTraitButton
         self.shouldGroupAccessibilityChildren = true
         
-        userImageView.user = ZMUser.selfUser()
+        if let imageData = self.account.imageData {
+            userImageView.imageView.image = UIImage(data: imageData)
+        }
+        else {
+            // TODO: internal protection level
+            //let personName = PersonName(name: self.account.userName ?? "",
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift]: `userName` is not exposed on Account?

Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/SelfProfile/AccountView.swift]: `PersonName`'s init is internal 🤕

**Comment by mikeger** [comment]
@typfel I updated the naming

**Pull request merged**